include/godot_cpp/variant/rect2.hpp

```diff
@@ -37,7 +37,7 @@
 
 namespace godot {
 
-struct Transform2D;
+class Transform2D;
```

Comment:
It's a struct, right? Why change this?

Reply:
No, it's defined as a class. Look up `Transform2D.hpp`, unless that is our real issue here.

Reply:
Why does `godot-cpp` use `class Transform2D` but the engine uses `struct`?

I apologize if this is off-topic, but this change seems very weird to me.

Reply:
To be honest, no idea, probably because we just generate everything as classes, @vnen is this something we should change?